From 13c3fb1b42223302b4a023e7b9a87535d042d187 Mon Sep 17 00:00:00 2001
From: Naushir Patuck <naush@raspberrypi.com>
Date: Thu, 16 Mar 2023 09:29:41 +0000
Subject: [PATCH 0960/1085] media: i2c: imx296: Adjust cropping limits

Through emperical testing, the sensor can crop upto a 96x88 window to
produce a valid Bayer frame. Adjust the ROIWH1_MIN ROIWV1_MIN
appropriately for this limit.

Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
---
 drivers/media/i2c/imx296.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/media/i2c/imx296.c
+++ b/drivers/media/i2c/imx296.c
@@ -154,9 +154,9 @@ MODULE_PARM_DESC(trigger_mode, "Set trig
 #define IMX296_FID0_ROIPH1				IMX296_REG_16BIT(0x3310)
 #define IMX296_FID0_ROIPV1				IMX296_REG_16BIT(0x3312)
 #define IMX296_FID0_ROIWH1				IMX296_REG_16BIT(0x3314)
-#define IMX296_FID0_ROIWH1_MIN				80
+#define IMX296_FID0_ROIWH1_MIN				96
 #define IMX296_FID0_ROIWV1				IMX296_REG_16BIT(0x3316)
-#define IMX296_FID0_ROIWV1_MIN				4
+#define IMX296_FID0_ROIWV1_MIN				88
 
 #define IMX296_CM_HSST_STARTTMG				IMX296_REG_16BIT(0x4018)
 #define IMX296_CM_HSST_ENDTMG				IMX296_REG_16BIT(0x401a)
